### PR TITLE
Feat/UI dom edits

### LINKED ISSUE #81: Check ability to tab through app
Add any issues for things that come up during this process

### LINKED ISSUE #98: After last merge, one line recipe titles have a weird glitchy effect.

### LINKED ISSUE #113: Recipe not being removed from the DOM after Remove from Saved button is clicked

### PR DESCRIPTION
**Pull request by qrispi**
Files Changed: index.html, scripts.js, styles.css, user.js, assets/images
Change Summary: Updated heart icon to match theme better. Added filter icon and re-designed filter drop-down menu to use a form with select options to enable full tab-ability. Fixed DOM error where user saved recipes were not being removed after un-saving. Re-enabled random user at page generation. Re-styled nav bar elements and added hover & focus classes. Added user message to blank saved recipes section. Re-named reduce iterator parameters. Resized and adjusted spacing on title logo. Removed glitchy title expansion on hover of recipe cards. Changed DOM population order of saved recipes to have most recently saved first. 

Closes #113, Closes #98, Closes #81

### CODE REVIEW

File: dist/index.html [hunk 1]
```diff
@@ -12,20 +12,22 @@
...
-                    <li>Main Course</li>
-                    <li>Salad</li>
-                    <li>Side Dish</li>
-                    <li>Dip</li>
-                    <li>Sauce</li>
-                 </div>
-            </div>
-            <li id="navMyRecipes" class="pointer" tabindex="0">My Recipes</li>
+            <input type="search" placeholder="Search Recipes..." class="input-bar search-bar hover" id="searchBar">
+            <form>
```
Comment (jfogiato) [inline dist/index.html:16]: Love the redesign here. Enables tabability much more easily and looks clean!

**Review by LordSchwifty — APPROVED** [review]
Nice job by the team figuring out how to remove recipe cards after un-saving them!

File: src/classes/User.js [hunk 2]
```diff
@@ -6,9 +6,10 @@ class User {
   }
 
   convertToFullRecipe(recipeRepo) {
-    return this.recipesToCook.map(userRecipe => {
-      return recipeRepo.recipes.find(recipe => recipe.id === userRecipe)
+    const fullRecipes = this.recipesToCook.map(userRecipe => {
+      return recipeRepo.recipes.find(recipe => recipe.id === userRecipe);
     });
+    return fullRecipes.reverse();
```
Comment (jfogiato) [inline src/classes/User.js:12]: Good thinking on getting saved recipes to generate earliest to latest!

**Review by jsahim — APPROVED** [review]
Really excellent work on this!

File: src/scripts.js [hunk 3]
```diff
@@ -44,162 +46,164 @@ searchBar.addEventListener('keyup', searchRecipes);
...
-        </div>
-    </div>`;
-    document.querySelector('#closeIcon').focus()
-    document.getElementById('saveBtn').addEventListener('click', toggleSaveRecipe);
-    document.getElementById('printBtn').addEventListener('click', () => window.print());
-    document.getElementById('closeIcon').addEventListener('keypress', (event) => {
-      event.key === "Enter" ? collapseRecipe(event) : null
-    });
-  }
+    if(event.target.className !== "all-recipes" || event.key === 'Enter') {
```
Comment (jfogiato) [inline src/scripts.js:79]: Looks like some weird formatting stuff? I'll try to make a pass tomorrow to refactor and clean up a bit

File: src/scripts.js [hunk 4]
```diff
@@ -44,162 +46,164 @@ searchBar.addEventListener('keyup', searchRecipes);
...
+    }
 }
 
-function filterRecipes(event) {
-    let tag = event.target.innerText.toLowerCase();
-    let filteredRecipes = savedView ? currentUser.filterSavedByTag(tag, recipeRepo) : recipeRepo.filterByTag(tag);
-    currentView = filteredRecipes;
-    createRecipeCards(currentView);
+function filterRecipes() {
+    const tag = document.getElementById('filters').value;
```
Comment (jfogiato) [inline src/scripts.js:136]: Good work leaning into the docs here! Glad we got this figured out.

File: dist/index.html (hunk 1 above)
Comment (jsahim) [reply dist/index.html:16]: This looks much sleeker!

File: src/scripts.js [hunk 5]
```diff
@@ -44,162 +46,164 @@ searchBar.addEventListener('keyup', searchRecipes);
...
+        const savedRecipes = currentUser.convertToFullRecipe(recipeRepo);
+        currentView = savedRecipes;
+        searchBar.placeholder = 'Search My Recipes...';
+        document.getElementById('filterPlaceholder').innerText = 'Filter My Recipes';
+        toggleHidden(myRecipesTitle);
+        toggleHidden(titleLogo);
+        createRecipeCards(currentView);
+        if(currentUser.recipesToCook.length < 1) {
+            recipeSection.innerHTML = "";
+            recipeSection.innerText = "Oh no - you haven't saved any recipes! Looks like you're going hungry tonight 🥲";
```
Comment (jfogiato) [inline src/scripts.js:194]: Love this :)

**Review by jfogiato — APPROVED** [review]
Great work all around! Glad we could figure out the few bugs we had!

**Pull request merged**